index.js

```diff
@@ -122,3 +129,9 @@ module.exports = function requireDir(dir, opts) {
 
     return map;
 };
+
+function toCamelcase(str) {
```

Comment:
Minor: ironically, this should be `toCamelCase`. =) For consistency with `String::toLower/UpperCase`.